FarsiLibrary.Win/FarsiLibrary.Win.csproj

```diff
@@ -12,7 +12,7 @@
     <AssemblyName>FarsiLibrary.Win</AssemblyName>
     <RootNamespace>FarsiLibrary.Win</RootNamespace>
     <OutputPath>..\binaries\</OutputPath>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
Why downgrade to v4.5?

Reply:
because of my project so i changed version